pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -149,6 +226,42 @@ func (c *clusterMemberRemovalController) sync(ctx context.Context, _ factory.Syn
 	return nil
 }
 
+func (c *clusterMemberRemovalController) removeMemberPendingDeletion(ctx context.Context, memberMachines []*machinev1beta1.Machine, memberType string) error {
+	memberMachinesPendingDeletion := ceohelpers.FilterMachinesPendingDeletion(memberMachines)
```

Comment:
this method is missing in this PR, is it DeletionTimestamp != nil?

Reply:
yes, https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/ceohelpers/common.go#L93